packages/x-data-grid/src/components/containers/GridRoot.tsx

```diff
@@ -91,4 +97,5 @@ GridRoot.propTypes = {
   ]),
 } as any;
 
-export { GridRoot };
+const GridRootMemo = React.memo(GridRoot);
```

Comment:
```suggestion
const MemoizedGridRoot = fastMemo(GridRoot);
```

To keep the same style as other parts of the codebase, with `fastMemo` from `@mui/x-internals`.